cmd/flux/get.go

```diff
@@ -100,17 +108,26 @@ func (get getCommand) run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	getAll := cmd.Use == "all"
+
 	if get.list.len() == 0 {
-		logger.Failuref("no %s objects found in %s namespace", get.kind, rootArgs.namespace)
+		if !getAll {
+			logger.Failuref("no %s objects found in %s namespace", get.kind, rootArgs.namespace)
```

Comment:
Wondering if it would be better to use `%q` instead of `%s` to highlight what the variables are.

Reply:
The kind is not an arg only the namespace is, I think the `%q` change has to be done all over the place not only here.

Reply:
Yes, agreed. I started doing it for the bootstrap commands, but we may indeed want to do one big sweep to settle it everywhere.